pkg/scaffold/project/project.go

```diff
@@ -19,39 +19,35 @@ package project
 import (
```

Comment:
Having a `PROJECT` file scaffold doesn't make sense if we have `config.New` and `c.Save`. @Adirio I assume you plan to reconcile this conflict in a follow-up?

Reply:
Exactly, the `Project` file scaffold is already an special case so the idea is to use `config.New` instead and thus avoiding all the overhead of templating that this file doesn't make use of. Actually this file and the boilerplate are being scaffolded first as they are required to scaffold the rest of the files relted to `kubebuilder init`.